src/functional.test.ts

```diff
@@ -639,8 +653,30 @@ describe('create-release-branch (functional)', () => {
           workspaces: {
             '.': ['packages/*'],
           },
+          createInitialCommit: false,
```

Comment:
Now that we confirm that changelogs exist before attempting to restore them, do we need these changes any longer?